src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor();
+        Random randomNumberSupplier = new Random();
+        int number = randomNumberSupplier.nextInt(101);

```

Comment:
All magic numbers in your code should be constants. The number '101' should be extracted into a constant with a meaningful name to avoid magic numbers in the code.